### DISCUSSION TITLE
2.14.0版本通过fid获取群公告出现bug

### DISCUSSION THREAD

**Discussion by yujumpjump** [mamoe/mirai#2517] [Q&A]
通过announcements.toList 可以获取到群里面所以的公告 但是 通过get(fid) 出现bug   kotlinx.serialization.MissingFieldException: Field 'msg' is required for type with serial name 'net.mamoe.mirai.internal.contact.announcement.GroupAnnouncement', but it was missing

这个bug 请问什么时候可以修复

**Comment by cssxsh** [comment]
https://github.com/mamoe/mirai/pull/2512

修复的代码已经提交了
可能在 2.14.1 合并

**Reply by yujumpjump** [reply]
好的 谢谢你们  麻烦你们快一点合并 麻烦了
